data/lua/core/wml.lua

```diff
@@ -189,14 +189,14 @@ if wesnoth.kernel_type() == "Game Lua Kernel" then
 	--- Calling wesnoth.fire isn't the same as calling wesnoth.wml_actions[name] due to the passed vconfig userdata
```

Comment:
This isn't called `wesnoth.fire` anymore.

Reply:
Oh, good point! I'll fix that.